can/parser_pyx.pyx

```diff
@@ -121,27 +121,26 @@ cdef class CANParser:
       cv_name = <unicode>cv.name
 
       self.vl[cv.address][cv_name] = cv.value
-      self.ts[cv.address][cv_name] = cv.ts
-
       self.vl[name][cv_name] = cv.value
-      self.ts[name][cv_name] = cv.ts
 
-      updated_val.insert(cv.address)
+      self.updated[cv.address][cv_name] = cv.updated_values
+      self.updated[name][cv_name] = cv.updated_values
+
+      if cv.updated_values.size():
+        updated_val.insert(cv.address)
 
     return updated_val
 
-  def update_string(self, dat, sendcan=False):
+  def update_string(self, dat, sendcan=False, update_vl=True):
```

Comment:
Since update_vl returns all values and sets updated vals, we don't want to reset the vals each string if updating multiple